pnpm-lock.yaml

```diff
@@ -283,7 +283,7 @@ importers:
       postcss-modules: 6.0.0_postcss@8.4.21
       resolve.exports: 2.0.0
       rollup-plugin-license: 3.0.1_rollup@3.10.0
-      sirv: 2.0.2_w6q35pvk7bmykgqf2hieut43iq
+      sirv: 2.0.2_hmoqtj4vy3i7wnpchga2a2mu3y
```

Comment:
I reverted pnpm-lock on my machine and ran `pnpm i` and pushed a commit of the update after running `pnpm i`. There was a inconsistency of hash on this line. (and on some other places)